gammapy/data/event_list.py

```diff
@@ -132,7 +132,14 @@ def altaz(self):
 
         lon, lat = self['AZ'], self['ALT']
         return SkyCoord(lon, lat, unit='deg', frame=altaz_frame)
-
+    
+    @property
+    def offset(self):
```

Comment:
Thanks for adding `EventList.offset`!
Please add a test for this new property at the end of the test_EventList function here:
https://github.com/gammapy/gammapy/blob/master/gammapy/data/tests/test_event_list.py#L12

Compute the offset, and assert on the value for one or two items.
The way I implement such asserts where the reference value isn't known is to put an arbitrary number for the expected value, run the test, get the actual value from the console output of the failed test, and put that in.

This is the best one can do for things where no externally verified expected values are available.
It's of course not perfect to do this, but it's useful to add that assert nonetheless, as a regression test so that we notice when results change on different machines / when refactoring code.